lib/solitude/api.py

```diff
@@ -27,10 +27,12 @@
 
 class BuyerNotConfigured(Exception):
     """The buyer has not yet been configured for the payment."""
+    solitude = 'BUYER_NOT_CONFIGURED'
```

Comment:
``solitude`` is a confusing attribute name to me. Could you call it ``error_code`` instead maybe?